bot/commands.php

```diff
@@ -49,3 +49,8 @@
         sendMessage($update['message']['chat']['id'], '<b>' . getTranslation('not_supported') . '</b>');
     }
 }
+// IF a Message type is private and there is no leading '/'
```

Comment:
Just to clarify, we're not evaluating a command but a reply to a command? 

Telegram has built in functionality for getting responses, could we not use that: https://core.telegram.org/bots/api#forcereply ?

Reply:
Really like that idea! @Artanicus 
This way we would not need a trainername_time database field at all 😁

Reply:
We're not really built for using forceReply efficiently so I'll do some plumbing in `telegram/functions.php` to generalize the `$inline_keyboard` to a `$reply_markup` array.